code/modules/chemistry/tools/balloons.dm

```diff
@@ -183,6 +183,10 @@
 					qdel(src)
 
 			if ("Inhale")
+				if (src.tied == TRUE)
```

Comment:
```suggestion
				if (src.tied)
```
Slightly prettier, if something is only ever `TRUE/FALSE` we prefer to just check `if (thing)`
(or `if (!thing)` if it should be FALSE)